tests/lax_numpy_test.py

```diff
@@ -4497,7 +4497,6 @@ def testWrappedSignaturesMatch(self):
       'eye': ['order'],
       'full': ['order'],
       'full_like': ['subok', 'order'],
-      'gradient': ['varargs', 'axis', 'edge_order'],
```

Comment:
Did `axis` become unsupported in this change?

Reply:
axis is still supported, but now is supported as an explicit keyword, rather than via ``**kwargs``

Reply:
I misread the diff! Got it now.